src/index.ts

```diff
@@ -47,8 +47,8 @@ export * from './lib/utils/preconditions/containers/UserPermissionsPrecondition'
 export * from './lib/utils/preconditions/IPreconditionContainer';
 export * from './lib/utils/preconditions/PreconditionContainerArray';
 export * from './lib/utils/preconditions/PreconditionContainerSingle';
+export * as CorePreconditions from './preconditions';
 export type { CooldownContext } from './preconditions/Cooldown';
-export { CorePrecondition as ClientPermissionsCorePrecondition } from './preconditions/ClientPermissions';
```

Comment:
Restore this please, because otherwise this would be a breaking change. 

Then add above the exports line:

```ts
/**
 * @deprecated. Please use `CorePreconditions.ClientPermissions`. `ClientPermissionsCorePrecondition` will be removed in v3.0.0
 */
```

Reply:
Yeah, I was thinking about doing that but didn't know what constituted as a breaking change. This would be an incredibly silly thing to bump to v3 over LMAO. 👍

Reply:
Basically anything that well.. could break someones code.. is a breaking change. If someone is using this import, they updated to the latest minor version (that till will be) either by choice or through caret semver and then run their bot it shouldn't crash because it can't find an import. That would happen when removing this, ergo it's a breaking change.